keps/prod-readiness/sig-network/3836.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3836
+alpha:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t I know this one is late, so if you say NAK, we will not argue :)

Reply:
I've read it today and I'm mostly fine with it.
I will just do another quick pass tomorrow to double check and approve.